src/components/tabs/_tabs.scss

```diff
@@ -110,12 +110,12 @@
         padding-left: govuk-spacing(4);
         border: 1px solid $govuk-border-colour;
         border-top: 0;
+
+        &.js-hidden {
+          display: none;
+        }
       }
     }
 
   }
-
-  .js-hidden {
```

Comment:
good news is the tests found this selector being defined globally

i imagine I should fix this behaviour in a different PR first...

Reply:
Makes me think having this class used with slightly different behaviours is indeed dangerous.

So we should either

1. Continue doing js-hidden for specific components but namespacing with their component names: `govuk-tabs-js-hidden`
2. Pull `js-hidden` out into a utility class that is used by tabs and conditional reveals

I feel 2. is probably the best way, since others can use this class too.

Reply:
Pushed up some scrappy working to see what that might look like, will do something clean next week.